pydantic/config.py

```diff
@@ -562,13 +562,15 @@ class Transaction(BaseModel):
     3. Using `'never'` we would have gotten `user=SubUser(hobbies=['scuba diving'], sins=['lying'])`.
     """
 
-    ser_json_timedelta: Literal['iso8601', 'float']
+    ser_json_timedelta: Literal['iso8601', 'milliseconds_float', 'seconds_float']
```

Comment:
```suggestion
    ser_json_timedelta: Literal['iso8601', 'seconds_float', 'milliseconds_float']
```

Nit pick, let's keep the order the same as that of `pydantic-core`